letsencrypt/tests/cli_test.py

```diff
@@ -43,7 +43,11 @@ def _call(self, args):
         return ret, stdout, stderr, client
 
     def test_no_flags(self):
-        self.assertRaises(SystemExit, self._call, [])
+        with mock.patch('letsencrypt.cli.run') as mock_run:
+            from letsencrypt import cli
+            cli.VERBS["run"] = mock_run
+            self._call([])
+            self.assertEqual(1, mock_run.call_count)
```

Comment:
CLI has been bugfixed a lot of times, often introducing regressions. I'm afraid that this will happen again, if we just don't provide tests :( In fact, how am I meant to test this PR? Test everything manually myself? It's really hard to verify just by looking at the code.